tasks/main.yml

```diff
@@ -1,19 +1,15 @@
 ---
 - name: Include OS-specific variables.
-  include_vars: "{{ ansible_os_family }}.yml"
+  include_tasks: variables.yml
```

Comment:
This is simple enough of an include that I'd rather just put the two tasks inline (instead of breaking them into a separate/new `variables.yml` file).